roles/maas_postgres_primary/tasks/teardown.yaml

```diff
@@ -1,11 +1,5 @@
 ---
 
-- name: "Install psycopg2 For Configuration"
-  ansible.builtin.pip:
-    name: "psycopg2"
-    version: "2.9.3"
-    state: "present"
-
 - name: "Remove Replication User"
   community.postgresql.postgresql_user:
```

Comment:
Actually, @cpg1111 are these tasks even necessary if we remove postgres in the next step?

Reply:
hmm not really, we could remove these altogether.

Reply:
excellent